ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_trigonometric_functions.py

```diff
@@ -311,3 +311,54 @@ def test_numpy_cosh(
         subok=True,
         test_values=False,
     )
+
+    
+# arctan2
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_float_dtypes,num_arrays=2),
+    dtype=st.sampled_from(ivy_np.valid_float_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    as_variable=helpers.array_bools(num_arrays=2),
+    with_out=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.arctan2"
+    ),
+    native_array=helpers.array_bools(num_arrays=2),
+)
+def test_numpy_arctan2(
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    input_dtype = [input_dtype]
```

Comment:
remove this line

Reply:
which one 339 ?

Reply:
really  I tried multiple times , through my trial I only passed one test (and the passed test varies)and error in lint decreased to 3 or 4 errors then after multiple trials the errors increased again and I can't find where is the problem , any thing else I can fix ?

Reply:
finally one test passed , lint errors descreased. but I'm afraid if I change code , lint error 'll increase again 

lint error are limited to those
ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py:1[8](https://github.com/unifyai/ivy/runs/8159720972?check_suite_focus=true#step:5:9)5:23: E231 missing whitespace after ','
ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py:1[9](https://github.com/unifyai/ivy/runs/8159720972?check_suite_focus=true#step:5:10)3:1: W391 blank line at end of file
ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_trigonometric_functions.py:319:84: E231 missing whitespace after ','
ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_trigonometric_functions.py:319:89: E501 line too long (98 > 88 characters)

Reply:
ivy/stateful/initializers.py:11 in public class `Initializer`:
        D204: 1 blank line required after class docstring (found 0) 
that's the only visible error in docstring test , how can I handle it